docs/sources/developers/plugins/_index.md

```diff
@@ -18,8 +18,8 @@ npx @grafana/toolkit plugin:create my-grafana-plugin
 
 If you want a more guided introduction to plugin development, check out our tutorials:
 
-- [Build a panel plugin]({{< relref "../../../../../tutorials/build-a-panel-plugin.md" >}})
-- [Build a data source plugin]({{< relref "../../../../../tutorials/build-a-data-source-plugin.md" >}})
+- [Build a panel plugin]({{< relref "../../../../../../tutorials/build-a-panel-plugin.md" >}})
```

Comment:
I copied the files based on how it's done in the drone scripts and for me the original pathing for this line seems correct.

Reply:
they look incorrect to me, and this fixes the build

Reply:
what drone script are you talking about? and where did you copy this? is this fixed in master already?